docs/workflows/create_workflow_block.md

```diff
@@ -0,0 +1,1898 @@
+
+        import supervision as sv
+        
+        from inference.core.workflows.execution_engine.entities.base import (
+            Batch,
+            OutputDefinition,
+            WorkflowImageData,
+        )
+        from inference.core.workflows.execution_engine.entities.types import (
+            BATCH_OF_IMAGES_KIND,
+            BATCH_OF_OBJECT_DETECTION_PREDICTION_KIND,
+            StepOutputImageSelector,
+            StepOutputSelector,
+            WorkflowImageSelector,
+        )
+        from inference.core.workflows.prototypes.block import (
+            BlockResult,
+            WorkflowBlock,
+            WorkflowBlockManifest,
+        )
+        
+        
+        class BlockManifest(WorkflowBlockManifest):
+            type: Literal["my_plugin/tile_detections@v1"]
+            images_crops: Union[WorkflowImageSelector, StepOutputImageSelector]
+            crops_predictions: StepOutputSelector(
+                kind=[BATCH_OF_OBJECT_DETECTION_PREDICTION_KIND]
+            )
+
+            @classmethod
+            def accepts_batch_input(cls) -> bool:
+                return True
+        
+            @classmethod
+            def get_output_dimensionality_offset(cls) -> int:
+                return -1
+        
+            @classmethod
+            def describe_outputs(cls) -> List[OutputDefinition]:
+                return [
+                    OutputDefinition(name="visualisations", kind=[BATCH_OF_IMAGES_KIND]),
+                ]
+        
+        
+        class TileDetectionsBlock(WorkflowBlock):
+        
+            @classmethod
+            def get_manifest(cls) -> Type[WorkflowBlockManifest]:
+                return BlockManifest
+        
+            def run(
+                self,
+                images_crops: Batch[Batch[WorkflowImageData]],
+                crops_predictions: Batch[Batch[sv.Detections]],
+            ) -> BlockResult:
+                annotator = sv.BoundingBoxAnnotator()
+                visualisations = []
+                for image_crops, crop_predictions in zip(images_crops, crops_predictions):
+                    visualisations_batch_element = []
+                    for image, prediction in zip(image_crops, crop_predictions):
+                        annotated_image = annotator.annotate(
+                            image.numpy_image.copy(),
+                            prediction,
+                        )
+                        visualisations_batch_element.append(annotated_image)
+                    tile = sv.create_tiles(visualisations_batch_element)
+                    visualisations.append({"visualisations": tile})
+                return visualisations
+        ```
+        
+        * lines `31-33` manifest that block is expected to take batches as input
+
+        * in lines `35-37` manifest class declares output dimensionality 
+        offset - value `-1` should be understand as decreasing dimensionality level by `1`
```

Comment:
`should be understand` -> `should be understood`

Reply:
👍